pyproject.toml

```diff
@@ -27,7 +27,7 @@ classifiers = [
 requires-python = ">=3.8"
 
 dependencies = [
-    "tutor>=18.0.0,<19.0.0"
+    "tutor>=16.0.0,<17.0.0"
```

Comment:
Should we change this [version](https://github.com/eduNEXT/drydock-backups/blob/hpg/palm-support/drydock_backups/__about__.py) value to start from 16? or is it automatic?

Reply:
It is automatic once we run the release